.github/workflows/publish.yaml

```diff
@@ -28,6 +29,5 @@ jobs:
         with:
           node-version: 22.x
           registry-url: https://registry.npmjs.org/
-      - run: npm publish
-        env:
-          NODE_AUTH_TOKEN: ${{secrets.npm_token}}
+      - run: npm install -g npm@latest

```

Comment:
A possible alternative to `npm install -g npm@latest` might be to use `node-version: 24`, which provides npm 11.6.0 and therefore support for trusted publishing. This approach should also help prevent breaking changes creeping in, e.g. it won't introduce npm 12.